src/h5web/metadata-viewer/MetadataViewer.tsx

```diff
@@ -13,9 +14,13 @@ interface Props {
 
 function MetadataViewer(props: Props) {
   const { path, onSelectPath } = props;
-
   const { entitiesStore } = useContext(ProviderContext);
-  const entity = entitiesStore.get(path);
+
+  const entity = handleError(
+    () => entitiesStore.get(path),
+    ProviderError.NotFound,
+    `No entity found at ${path}`
+  );
```

Comment:
I'm open to naming suggestions for `handleError` if you're not a fan.

Inlining the `try`/`catch` feels very repetitive and complex, as it requires the use of `let entity`:

```ts
let entity;
try {
  entity = entitiesStore.get(path);
} catch (...) {
  ...
}

// do something with `entity`
```